package.json

```diff
@@ -38,9 +40,12 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "prepublish": "npm test",
+    "prepublish": "npm run check-links && npm test",
     "test": "npm run lint && node bin/jest.js && npm run jasmine1",
-    "jasmine1": "node bin/jest.js --testRunner='<rootDir>/testRunners/jasmine/jasmine1.js'"
+    "jasmine1": "node bin/jest.js --testRunner='<rootDir>/testRunners/jasmine/jasmine1.js'",
+    "check-links": "[ \"$NODE_ENV\" != production ] && npm run link-packages",
```

Comment:
why are you checking for node env here? :)

I think `npm test` should also do `check-links`.